app/models/incident_contact.rb

```diff
@@ -0,0 +1,7 @@
+class IncidentContact < ApplicationRecord
+  belongs_to :incident
+  belongs_to :contact
+
+  validates_presence_of :incident_id, :contact_id
```

Comment:
May need to add in here with the update to the `incidents_contacts` table that it `validates_presence_of :title`